### PR TITLE
Newtests week49

### PR DESCRIPTION
**Pull request by bcarrez**
New batch of tests from DEFROST team. This branch only contains new tests, no new feature.

### CODE REVIEW

File: SofaKernel/framework/framework_test/CMakeLists.txt [hunk 1]
```diff
@@ -14,6 +14,8 @@ set(SOURCE_FILES
     helper/Quater_test.cpp
     helper/SVector_test.cpp
     helper/io/MeshOBJ_test.cpp
+    helper/io/MeshSTL_test.cpp
+    helper/io/ImagePNG_test.cpp
```
Comment (guparan) [inline SofaKernel/framework/framework_test/CMakeLists.txt]: Conflict resolution went wrong here. We should not have ImagePNG_test on line 18.
See 67f714f3

**Review by guparan — CHANGES_REQUESTED** [review]

File: SofaKernel/framework/framework_test/CMakeLists.txt (hunk 1 above)
Comment (damienmarchal) [reply SofaKernel/framework/framework_test/CMakeLists.txt]: Thanks for the feedback @guparan  
I will fix that asap.

**Comment by damienmarchal** [comment]
@guparan Done

**Comment by matthieu-nesme** [comment]
hooray to the defrost team ;)

**Review by guparan — APPROVED** [review]

**Comment by guparan** [comment]
Could you please manually rebase your branch to the updated upstream/master? Some changes on master prevent GitHub from auto-rebasing.
```
git checkout newtests_week49
git fetch upstream/master
git rebase upstream/master
```

**Pull request merged**